generalInfoPage/general.css

```diff
@@ -21,7 +21,7 @@ header {
   font-size: 1.5rem;
 }
 
-.menu-icon {
+.menuIcon {
```

Comment:
can't use two different convention for naming in same time and same file
change it to be .menu-icon